autogluon/utils/tabular/ml/models/ensemble/stacker_ensemble_model.py

```diff
@@ -218,13 +219,13 @@ def hyperparameter_tune(self, X, y, k_fold, scheduler_options=None, compute_base
         # TODO: hpo_results likely not correct because no renames
         return stackers, stackers_performance, hpo_results
 
-    def load_base_model(self, model_name):
+    def load_base_model(self, model_name, compression_fn=None, compression_fn_kwargs=None):
```

Comment:
consider replacing with **kwargs to send to the model_type.load(model_path, **kwargs)